src/components/HomeNavBar.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import MenuIcon from '@material-ui/icons/Menu';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import PersonIcon from '@material-ui/icons/Person';
+import BuildIcon from '@material-ui/icons/Build';
+import ShoppingCartIcon from '@material-ui/icons/ShoppingCart';
+import ReportProblemIcon from '@material-ui/icons/ReportProblem';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import WbSunnyIcon from '@material-ui/icons/WbSunny';
+import { Root, Bar, MenuButton, AccountButton, ListStyle, ToolBar, Typograph } from '../styles/TopNavStyles';
```

Comment:
You can import all of the styled components at once by using `import * as Styles from ..` and then prefacing each of your styled components with `Styles.`

You can see how Tiffany did that [here](https://github.com/calblueprint/meepanyar/blob/8d6b89b238464e920da5184c6371a3d75faae0c5/src/screens/Login.tsx)!